lib/commands/general.js

```diff
@@ -234,7 +234,18 @@ commands.mobileViewportScreenshot = async function () {
 commands.setUrl = async function (url) {
   await this.adb.startUri(url, this.opts.appPackage);
 };
+/**
+ * @typedef {object} DeepLinkOpts
+ * @property {?string} url - The name of URL to start.
```

Comment:
the type would be `{!string}` since this property is required

Reply:
Oh! Yes.